### PR TITLE
Include option to disable noisy POSIX warning in Annotations

### LINKED ISSUE #32: Noisy POSIX warning
Hi, i'm getting a lot of warnings from your action like "One of the secrets has a name that is not POSIX compliant...", because basically all my secrets are not POSIX compliant. But since the conversion behavior is already documented in the readme and works fine i think it would be ok to use a debug message instead of a warning, this way it can still help to analyze and track down issues but doesn't clutter the workflow logs.

### PR DESCRIPTION
**Pull request by jbabala-costa**
Fixes #32

### CODE REVIEW

**Review by antonycohen — APPROVED** [review]
Seem perfect to me

File: README.md [hunk 1]
```diff
@@ -64,6 +64,9 @@ steps:
 | `true`       | `foo` = `{ "bar": "baz" }`<br>`ham` = `eggs`     | `FOO_BAR` = `baz` AND<br>`ham` = `eggs`             | If multiple secrets, values that can be parsed into a JSON will be parsed and flattened |
 | `false`      | `dev_foo` = `{ "bar": "baz" }`                   | `DEV_FOO` = `{ "bar": "baz" }`                      | Not parsed                                                                              |
 
+- `disable-warnings`
+  - If `disable-warnings: true`. the annotation warnings regarding posix compliance in GitHub Actions output will be supressed.
```
Comment (abhilash1in) [inline README.md:68]:
```suggestion
  - If `disable-warnings: true`, the annotation warnings regarding POSIX compliance in GitHub Actions output will be supressed.
```

**Review by abhilash1in — COMMENTED** [review]
Thanks for the PR! Looks good overall. Added a minor comment. Also consider `suppress-warnings` instead of `disable-warnings` (I don't have a strong preference of one over the other).

File: README.md (hunk 1 above)
Comment (abhilash1in) [reply README.md:68]: Also, what's an `annotation warning`? Does it mean anything different from a simple `warning`? Can we simplify it in the below way?
```suggestion
  - If `disable-warnings: true`, warnings regarding POSIX compliance in GitHub Actions output will be suppressed.
```

**Comment by holooloo** [comment]
Please merge it

**Review by abhilash1in — APPROVED** [review]
Looks good, thanks!

**Pull request merged**